tests/scripts/run-throughput-perf.py

```diff
@@ -250,7 +287,7 @@ def runIterations(dll_name, dll_path, iterations, crossgen_path, jit_path, assem
     # Time.clock() returns seconds, with a resolution of 0.4 microseconds, so multiply by the multiplier to get milliseconds
     multiplier = 1000
 
-    for iteration in range(0,iterations):
+    for iteration in range(0,iterations + 1):
```

Comment:
Going backwards from my earlier question: if iterations is a string, does "+ 1" work for it? (Or is it an int?)